cql/src/main/java/org/apache/cassandra/stargate/transport/internal/messages/AuthResponse.java

```diff
@@ -72,7 +72,11 @@ public AuthResponse(byte[] token) {
       byte[] challenge = negotiator.evaluateResponse(token);
       if (negotiator.isComplete()) {
         AuthenticatedUser authenticatedUser = negotiator.getAuthenticatedUser();
-        persistenceConnection().login(authenticatedUser);
+        if (!(authenticatedUser.usesTransitionalAuth()
+            && Boolean.parseBoolean(
+                System.getProperty("stargate.cql_use_transitional_auth", "false")))) {
```

Comment:
we should add some documentaiton around: `"stargate.cql_use_transitional_auth`

Reply:
We need to add consolidated documentation for all of these flags

Reply:
Could we also add an integration test for `stargate.cql_use_transitional_auth` == `true`?